### PR TITLE
Master mrp usability imp hsh

### PR DESCRIPTION
**Pull request by hsh-odoo**
Description of the issue/feature this PR addresses:
Task : https://www.odoo.com/web#id=28941&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Pad : https://pad.odoo.com/p/r.74d44c58dc7c0241a07f9764eeaf0d41

Current behavior before PR:
- usability improvements in MRP

Desired behavior after PR is merged:
- Way to still make invisible update link when creating an MO (did not manage to do so)
- Workorder messages menu should not be visible when no workorders activated (put groups on menuitem)
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mrp/views/mrp_production_views.xml [hunk 1]
```diff
@@ -90,7 +91,7 @@
                                 <field name="product_uom_id" options="{'no_open':True,'no_create':True}" groups="product.group_uom" attrs="{'readonly': [('has_moves', '=', True)]}"/>
                                 <button type="action"
                                     name="%(mrp.action_change_production_qty)d"
-                                    string="Update" states="confirmed,planned,progress" class="oe_link" attrs="{'invisible': [('state','=','done')]}"/>
+                                    string="Update" states="planned,progress" class="oe_link" attrs="{'invisible': ['|', ('state', '=', 'done'), ('id', '=', False)]}"/>
```
Comment (jco-odoo) [inline addons/mrp/views/mrp_production_views.xml]: Why did you remove confirmed?

Comment (hsh-odoo) [inline addons/mrp/views/mrp_production_views.xml]: If we want to show or hide "update link" based on some condition, condition will not work for provided states (in the field attribute), so we need to remove particular state ("confirmed") from the states (field attribute)

**Comment by jco-odoo** [comment]
But this means that if you save a manufacturing order and you have not
planned it yet, you won't be able to update it?

File: addons/mrp/views/mrp_production_views.xml (hunk 1 above)
Comment (tde-banana-odoo) [inline addons/mrp/views/mrp_production_views.xml]: Actually mixing states attribute and setting invisible based on state makes it quite difficult to understand. Please use only attrs.

**Comment by jco-odoo** [comment]
Ok for merge to me.  @sle-odoo
